docs/Deep Learning/Multilayer Perceptron (MLP).md

```diff
@@ -0,0 +1,129 @@
+---
+id: Multilayer Perceptron (MLP) in Deep Learning
```

Comment:
replace `id: Multilayer Perceptron (MLP) in Deep Learning` with `id: multilayer-perceptron-in-deep-learning`